documentation/views.py

```diff
@@ -42,16 +44,20 @@ def embed_view(request, slug, ide_slug):
     return render(request, 'documentation/embed.html', {'code_block': block, 'ide': ide})
 
 
-def page_view(request, slug, curric_slug):
-    try:
-        page = Map.objects.get(slug=slug, parent__slug=curric_slug)
-    except Map.DoesNotExist:
-        raise ContinueResolving
+def page_view(request, parents, slug):
+    pages = Map.objects.filter(slug=slug)
+
+    if pages.count() > 1 and parents is not None:
+        # If more than one map was found with the same slug, try narrowing by parent
+        parent_slug = parents.split('/')[-1]
```

Comment:
any reason in particular we're only using a single parent while also allowing for an arbitrary number in the url? Seems like we should only do one or the other

Reply:
My thinking was that sibling slugs should be unique, so if we run into a case where more than one map shows up for a given slug filtering by the immediate parent should catch it. I could certainly make it walk up the parent ladder if the duplication isn't resolved by the immediate parent.

Reply:
What do you think about actually enforcing sibling uniqueness with a `unique_together = ("parent", "slug")`?

Reply:
In which case we would always query by map and parent slugs here?

Reply:
Yep!

Reply:
Oh dang, I didn't realize Django built derived classes that way.

It's probably not critical to solve this, but it makes me a little uncomfortable to have this potential for unexpected behavior just hanging out.

What do you think about throwing a 500 error in the case where `pages.count() > 1`, with an error message that explains all of this and how to fix it (ie by going into the admin panel and making things appropriately unique)?